shell/platform/windows/compositor_opengl.cc

```diff
@@ -50,8 +51,32 @@ bool CompositorOpenGL::CreateBackingStore(
                   GL_UNSIGNED_BYTE, nullptr);
   gl_->BindTexture(GL_TEXTURE_2D, 0);
 
-  gl_->FramebufferTexture2D(GL_FRAMEBUFFER, GL_COLOR_ATTACHMENT0, GL_TEXTURE_2D,
-                            store->texture_id, 0);
+  if (enable_impeller_) {
+    gl_->FramebufferTexture2DMultisampleEXT(GL_FRAMEBUFFER,
```

Comment:
Could we add a very quick comment that explains why this needs to be different for Impeller?